Userland/Applications/FileManager/PropertiesWindow.cpp

```diff
@@ -251,6 +257,98 @@ ErrorOr<void> PropertiesWindow::create_audio_tab(GUI::TabWidget& tab_widget, Non
     return {};
 }
 
+ErrorOr<void> PropertiesWindow::create_image_tab(GUI::TabWidget& tab_widget, NonnullRefPtr<Core::MappedFile> mapped_file, StringView mime_type)
+{
+    auto image_decoder = Gfx::ImageDecoder::try_create_for_raw_bytes(mapped_file->bytes(), mime_type);
+    if (!image_decoder)
+        return {};
+
+    auto tab = TRY(tab_widget.try_add_tab<GUI::Widget>("Image"_short_string));
+    TRY(tab->load_from_gml(properties_window_image_tab_gml));
+
+    // FIXME: The JPEG decoder doesn't decode its header (and so, know what its size is) until we request either a frame,
```

Comment:
Can you file an issue for this?

I noticed this too at some point while writing tests (in cee1f9b I think – if the first two lines in expect_single_frame_of_size() are swapped, jpeg tests fail, and they really shouldn't). I should've made an issue for that back then.